src/main/java/DivideByZero.java

```diff
@@ -0,0 +1,11 @@
+// https://sig-product-docs.synopsys.com/bundle/coverity-docs/page/checker-ref/checkers/D/divide_by_zero.html
+
+class DivideByZero {
+    void testDiv(int a, int b)
+    {
+        if (a!=0) {
+           //Do something
+        }
+        int y = b / a;
```

Comment:
#### Automated PR Comment From coverity
<!-- 6f7fb9037861827363423baa19a14521 -->
<!-- Comment managed by Git Commenter, do not modify! -->
### Coverity Issue - DLS: Dead local store
_Low CWE-563_
Dead store to y.